Signum.Engine.Extensions/Mailing/EmailLogic.cs

```diff
@@ -39,13 +39,13 @@ internal static void AssertStarted(SchemaBuilder sb)
         }
 
         public static void Start(
-            SchemaBuilder sb,  
-            Func<EmailConfigurationEmbedded> getConfiguration, 
-            Func<EmailTemplateEntity?, Lite<Entity>?, EmailSenderConfigurationEntity> getEmailSenderConfiguration,  
+            SchemaBuilder sb,
+            Func<EmailConfigurationEmbedded> getConfiguration,
+            Func<EmailTemplateEntity?, Entity?, EmailSenderConfigurationEntity> getEmailSenderConfiguration,

```

Comment:
Please change to Func<EmailTemplateEntity?, EmailMessage?, Lite<Entity>?, EmailSenderConfigurationEntity>